external/libtww/d/save/d_save.h

```diff
@@ -262,15 +262,22 @@ static_assert(sizeof(dSv_player_item_max_c) == 0x8);
 
 class dSv_player_bag_item_c {
 public:
-    /* 0x00 */ u8 field_0x0[8];
-    /* 0x08 */ u8 field_0x8[8];
-    /* 0x10 */ u8 field_0x10[8];
+    u8 getDeliveryBagSlot(u8 index) { return mDeliveryBag[index]; }
+    void setDeliveryBagSlot(u8 index, u8 has_item) { mDeliveryBag[index] = has_item; }
+    u8 getSpoilsBagSlot(u8 index) { return mSpoilsBag[index]; }
+    void setSpoilsBagSlot(u8 index, u8 has_item) { mSpoilsBag[index] = has_item; }
+
+    /* 0x00 */ u8 mSpoilsBag[8];
+    /* 0x08 */ u8 mBaitBag[8];
+    /* 0x10 */ u8 mDeliveryBag[8];
 };  // Size: 0x18
 
 static_assert(sizeof(dSv_player_bag_item_c) == 0x18);
 
 class dSv_player_get_bag_item_c {
 public:
+    u8 getSpoilFlag(u8 index) { return mBeastFlags; }
+    void setSpoilFlag(u8 collected_item) { mBeastFlags |= (1 << collected_item); }

```

Comment:
"mBeastFlags" should get renamed to "mSpoilsFlags" since that is what they actually are.  I'm guessing this is called "mBeastFlags" now because it was copied over from TP

Reply:
yes that was strange to me i can change that